### ISSUE TITLE
cannot format jaraco.classes.ancestry

### ISSUE THREAD

**Issue by jaraco** [psf/black#1772]
**Describe the bug** A clear and concise description of what the bug is.

Running `black` on jaraco/jaraco.classes@a5ee6c53463cc1d468cde1784de70c862e3989c2 fails with:

```
error: cannot format /Users/jaraco/code/main/jaraco.classes/jaraco/classes/ancestry.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/blk_wez_xks5.log
```

Log says:

```diff
--- src
+++ dst
@@ -192,11 +192,11 @@
           value=
             Constant(
               kind=
                 None,  # NoneType
               value=
-                "Generator over all subclasses of a given class, in depth-first order.\n\n>>> bool in list(iter_subclasses(int))\nTrue\n>>> class A(object): pass\n>>> class B(A): pass\n>>> class C(A): pass\n>>> class D(B,C): pass\n>>> class E(D): pass\n>>>\n>>> for cls in iter_subclasses(A):\n...\t\tprint(cls.__name__)\nB\nD\nE\nC\n>>> # get ALL classes currently defined\n>>> res = [cls.__name__ for cls in iter_subclasses(object)]\n>>> 'type' in res\nTrue\n>>> 'tuple' in res\nTrue\n>>> len(res) > 100\nTrue",  # str
+                "Generator over all subclasses of a given class, in depth-first order.\n\n>>> bool in list(iter_subclasses(int))\nTrue\n>>> class A(object): pass\n>>> class B(A): pass\n>>> class C(A): pass\n>>> class D(B,C): pass\n>>> class E(D): pass\n>>>\n>>> for cls in iter_subclasses(A):\n...         print(cls.__name__)\nB\nD\nE\nC\n>>> # get ALL classes currently defined\n>>> res = [cls.__name__ for cls in iter_subclasses(object)]\n>>> 'type' in res\nTrue\n>>> 'tuple' in res\nTrue\n>>> len(res) > 100\nTrue",  # str
             )  # /Constant
         )  # /Expr
         Return(
           value=
             Call(
```

**To Reproduce** Steps to reproduce the behavior:

1. git clone https://github.com/jaraco/jaraco.classes
2. cd jaraco.classes
3. git checkout a5ee6c53463cc1d468cde1784de70c862e3989c2
4. black .

**Environment (please complete the following information):**

- Version: 20.8b1
- OS and Python version: macOS 10.15

**Does this bug also happen on master?** To answer this, you have two options:

Yes

```
jaraco.classes master $ pip-run -q git+https://github.com/psf/black -- -m black .
error: cannot format /Users/jaraco/code/main/jaraco.classes/jaraco/classes/ancestry.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/blk_c3ygngeb.log
Oh no! 💥 💔 💥
6 files left unchanged, 1 file failed to reformat.
```

**Comment by jaraco** [comment]
I was able to work around the issue with jaraco/jaraco.skeleton@feea0b5. I'll leave it up to the maintainers whether this edge case is worth fixing.

**Comment by hauntsaninja** [comment]
Seems like a dupe of the reportedly fixed #1601

**Comment by ichard26** [comment]
Can confirm this is fixed by commit 1d8b4d7. I'm on Ubuntu 20.04.01 LTS and using CPython 3.8.5. 

I'm not sure whether this counts as a duplicate of #1601 so I'll just close this without attaching a duplicate label. Thanks for the heads up @hauntsaninja and the bug report report @jaraco!

**Comment by jaraco** [comment]
I believe it is a duplicate. I have encountered it a couple of times since in other projects.
